interface/forms/LBF/new.php

```diff
@@ -1704,7 +1709,11 @@ function warehouse_changed(sel) {
             ?>
             <br />
 
-            <div class="col-12">
+        </div><!-- end column -->
+    </div><!-- end row -->
+    <div class="row"><!-- was flex-row -->
+        <div class='col-12'>
```

Comment:
div block containing form tag is closed before form contents are complete through L-1805.
Pretty sure this will effect what's included on submit.

I'd replace where the form tag is echoed to a html tag with arg replacements being echoed. Will allow IDE to help with tag closures.
Here's a screen showing html error.
![image](https://user-images.githubusercontent.com/1859985/116015369-f677e800-a606-11eb-9c3b-4124abfaddb7.png)

Reply:
Not sure I understand about the HTML tag, but I moved the form open and close tags to be at the outer level, containing all the divs. Also removed form-inline from genProviderSelect() as you suggested. Testing OK here. Thanks for the insight and testing, I really appreciate it!

Reply:
@sunsetsystems 
Do we really want the dark background on all LBF's?
![image](https://user-images.githubusercontent.com/1859985/116143425-224baa00-a6a9-11eb-8245-9a5fe677a220.png)

It's always bothered me. I can change the theme builds to take the background out so LBF follows theme color.
![image](https://user-images.githubusercontent.com/1859985/116144105-eebd4f80-a6a9-11eb-8776-be9ee0b0d739.png)

I have pending PR I can add to....

Reply:
i like the contrast, :2nd_place_medal:

Reply:
It looks much better in the other colored themes so, maybe i'll pick on something else.:)